distribution/lib/Standard/Google_Api/0.0.0-dev/src/Internal/Google_Analytics_Internal.enso

```diff
@@ -14,13 +15,13 @@ polyglot java import com.google.analytics.data.v1beta.Metric
 polyglot java import com.google.analytics.data.v1beta.RunReportRequest
 
 read_api_data property_id:Text dimensions:Vector metrics:Vector start_date:Date end_date:Date credentials:Google_Credential -> Table =
-    analytics_data = case credentials of 
-        Google_Credential.Default -> BetaAnalyticsDataClient.create
+    managed_analytics_data = case credentials of
+        Google_Credential.Default -> Managed_Resource (BetaAnalyticsDataClient.create) r-> r.close
         Google_Credential.From_File _ -> 
             betaAnalyticsDataSettings = BetaAnalyticsDataSettings.newBuilder
                . setCredentialsProvider credentials.as_java
                . build
-            BetaAnalyticsDataClient.create betaAnalyticsDataSettings
+            Managed_Resource.register (BetaAnalyticsDataClient.create betaAnalyticsDataSettings) r-> r.close
```

Comment:
...this line is using `Managed_Resource.register` while line 19 only `Managed_Resource`.